src/components/Product.js

```diff
@@ -1,46 +1,44 @@
-import React, { Component } from "react";
+import React from "react";
 import rokok from "../assets/image/rokok.png";
 import styled from "styled-components";
 
-export default class Product extends Component {
-  render() {
-    const Image = styled.img`
-      width: 200px;
-      height: 250px;
-      min-height: 250px;
-      `;
+const Image = styled.img`
+  width: 200px;
+  height: 250px;
+  min-height: 250px;
+`;
 
-    const Product = styled.li`
-      width: 200px;
-      height: 300px;
-      margin: 30px;
-      display: flex;
-      flex-direction: column-reverse;
-     
-      margin: 30px;
-      justify-content: center;
-      margin-bottom: 20px;
-      `;
+const Wrapper = styled.li`
+  width: 200px;
+  height: 300px;
+  margin: 30px;
+  display: flex;
+  flex-direction: column-reverse;
 
-    const Title = styled.h1`
-      text-align: center;
-      min-height: 150px;
-      padding-top: 20px;
-      padding-bottom: 20px;
-      font-size: large;
-    `;
+  margin: 30px;
+  justify-content: center;
+  margin-bottom: 20px;
+`;
 
-    return (
-      <Product>
-        <Title>
-            <b>{this.props.name}</b>
-        </Title>
-        <Image
-          src={this.props.image == null ? rokok : this.props.image}
-          onClick={this.props.onClick}
-          alt="product"
-          />
-      </Product>
-    );
-  }
+const Title = styled.h1`
+  text-align: center;
+  min-height: 150px;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  font-size: large;
+`;
+
+export default function Product({ name, image, onClick }) {
+  return (
+    <Wrapper>
+      <Title>
+        <b>{name}</b>
+      </Title>
+      <Image
+        src={image == null ? rokok : image}
```

Comment:
What is "rokok"?

Reply:
It's a best practice to always use the `===` equality operator. See also: https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons.